src/main/resources/openapi/audiorecording.yaml

```diff
@@ -0,0 +1,130 @@
+openapi: 3.0.1
+info:
+  title: Audio Recording Service
+  version: v0.1.0
+servers:
+  - url: http://localhost:8080
+paths:
+  /audios:
+    post:
+      tags:
+        - audio-recording
+      summary: Upload an audio recording
+      operationId: addAudio
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AddAudioRequest'
+      responses:
+        200:
+          description: Recording accepted
+        400:
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetail'
+              examples:
+                'Specified court house not found':
+                  value:
+                    status: 400
+                    type: 'courthouse-not-found'
+                'Audio not provided':
+                  value:
+                    status: 400
+                    type: 'audio-not-provided'
+                'Unexpected file type':
+                  value:
+                    status: 400
+                    type: 'unexpected-file-type'
+                'Audio file size outwith allowable bounds':
+                  value:
+                    status: 400
+                    type: 'filesize-out-of-bounds'
+        500:
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetail'
+              examples:
+                'Failure to write to inbound data store':
+                  value:
+                    status: 500
+                    type: 'inbound-datastore-failure'
```

Comment:
For now these 4 errors don't make sense as no binary audio data is being sent in the request.

- Audio not provided
- Unexpected file type
- Audio file size outwith allowable bounds
- Failure to write to inbound data store

Suggest these be removed from the spec to avoid confusion. If the decision is made to include the binary data in the the request they can be reinstated as a future ticket.

Reply:
ACCEPT